### PR TITLE
#250 - add command for giving keys to office and taking them

### LINKED ISSUE #250: Add command for giving keys to/taking keys from office
Now we have three commands for keys:

- toby klucze - shows all keys
- toby klucze:od {user} - takes key form {user}
- toby klucze:do {user} - gives key to {user}

But we don't have commands for giving keys to/taking keys from office. It will be nice feature to make this. I have no idea how to call this commands, so I'm open to suggestions.

### PR DESCRIPTION
**Pull request by EwelinaSkrzypacz**
This should close #250.

Also I fixed bug with command `/toby dzisiaj` - now when the user types the command in Slack it will work.

### CODE REVIEW

File: app/Infrastructure/Slack/Handlers/LeaveKeysInOffice.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    /**
+     * @throws UserNotFoundException
+     * @throws ValidationException
+     */
+    public function handle(Request $request): Response
+    {
+        $authUser = $this->findUserBySlackIdOrFail($request->userId);
+
+        /** @var Key $key */
```
Comment (krzysztofrewak) [inline app/Infrastructure/Slack/Handlers/LeaveKeysInOffice.php:30]:
```suggestion
        /** @var Key|null $key */
```

File: app/Infrastructure/Slack/Handlers/TakeKeysFromOffice.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    /**
+     * @throws UserNotFoundException
+     * @throws ValidationException
+     */
+    public function handle(Request $request): Response
+    {
+        $authUser = $this->findUserBySlackIdOrFail($request->userId);
+
+        /** @var Key $key */
```
Comment (krzysztofrewak) [inline app/Infrastructure/Slack/Handlers/TakeKeysFromOffice.php:30]:
```suggestion
        /** @var Key|null $key */
```

File: app/Infrastructure/Slack/Handlers/LeaveKeysInOffice.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+use Toby\Domain\Notifications\KeyHasBeenLeftInTheOffice;
+use Toby\Eloquent\Models\Key;
+use Toby\Infrastructure\Slack\Exceptions\UserNotFoundException;
+use Toby\Infrastructure\Slack\Traits\FindsUserBySlackId;
+
+class LeaveKeysInOffice extends SignatureHandler
+{
+    use FindsUserBySlackId;
+
+    protected $signature = "toby klucze:biuro:zostaw";
```
Comment (krzysztofrewak) [inline app/Infrastructure/Slack/Handlers/LeaveKeysInOffice.php:19]: Why in Polish?

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]

File: app/Infrastructure/Slack/Handlers/LeaveKeysInOffice.php (hunk 3 above)
Comment (EwelinaSkrzypacz) [reply app/Infrastructure/Slack/Handlers/LeaveKeysInOffice.php:19]: Because we have all commands in Polish.

**Review by krzysztofrewak — APPROVED** [review]

**Review by Baakoma — APPROVED** [review]

**Pull request merged**